static/translations/messages_en.json

```diff
@@ -231,5 +231,12 @@
   "Signed out": "Signed out",
   "Remove account?": "Remove account?",
   "This will make the mobile app on this device forget {email} on {realmUrl}.": "This will make the mobile app on this device forget {email} on {realmUrl}.",
-  "Remove account": "Remove account"
+  "Remove account": "Remove account",
+  "Deactivate your account?": "Deactivate your account?",
+  "By deactivating your account, you will be logged out immediately.\n\nNote that any bots that you maintain will be disabled.": "By deactivating your account, you will be logged out immediately.\n\nNote that any bots that you maintain will be disabled.",
+  "Deactivate": "Deactivate",
+  "Deactivate your account {email} on: {realmUrl} from Zulip.": "Deactivate your account {email} on: {realmUrl} from Zulip.",
+  "Log you out immediately from Zulip, that includes Zulip's website, mobile, desktop, and terminal.":"Log you out immediately from Zulip, that includes Zulip's website, mobile, desktop, and terminal.",
```

Comment:
nit: Colon should be followed by a space to maintain consistency 🙂

Reply:
I'm not sure what you mean, can you be more specific? like where this rule isn't applying?

Reply:
Kindly check line number 239.
Note - There should be a blank space after the colon.

Reply:
aaaaaah, I see it now, thanks!